test/Tests/src/Data/Vector_Spec.enso

```diff
@@ -165,9 +168,21 @@ spec = Test.group "Vectors" <|
         txtvec.filter (Filter_Condition.Between "b" "c") . should_equal ["bbb", "baaa"]
         Test.expect_panic_with (txtvec.filter (Filter_Condition.Starts_With 42)) Unsupported_Argument_Types_Data
 
+        ["", Nothing, " ", "a"].filter (Filter_Condition.Is_Empty) . should_equal ["", Nothing]
+        ["", Nothing, " ", "a"].filter (Filter_Condition.Not_Empty) . should_equal [" ", "a"]
+        ["abab", "aaabaaaa", "ba"].filter (Filter_Condition.Like "ba") . should_equal ["ba"]
```

Comment:
@jdunkerley this is the test that verifies that our LIKE matches the whole string if no wildcards are added. We don't need ^$ because we do the regex `matches` check which itself is meant to check if the _whole_ text matches the pattern. We'd need ^$ if we used something like `search` instead

Reply:
great - thanks just wanted to check.